PROJECT-BRIEF.md

```diff
@@ -1,59 +1,186 @@
+
+### Week 2, {DATES_WEEK_2}
+
+| 🤝 Developer pairings | 🖥 Assigned Github issue                                                                                              |
+| --------------------- | --------------------------------------------------------------------------------------------------------------------- |
+| {DEV_2} & {DEV_3}     | [4. As a user, I want to set up a new shopping list so I can start tracking purchased items.]({REPO_URL}/issues/4)    |
+| {DEV_1} & {DEV_4}     | [5. As a user, I want to add a new item to my shopping list so I can start recording purchases.]({REPO_URL}/issues/5) |
+
+| 📖 Learning modules                                                                                                                                           | Description |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------- |
+| [Accessibility](https://youtu.be/xrP2DIoZRnQ) ([slides](https://docs.google.com/presentation/d/1ux4HXGj3XAGZRD7qvOpjl36FYbTcUYGaaBHywc95taY/edit#slide=id.p)) |
+
+| 👋 Sync meetings | Agenda                                                                                                                                                            |
+| ---------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Weekly Sync      | - Demo issues #2 & #3 <br> - Discuss Learning Modules <br> - Collabies will be presented with issue #4 and issue #5                                               |
+| Pair Programming | Schedule a pairing session(s) with your partner early in the week to complete the development work required to meet the acceptance criteria of your Github issue. |
+| Optional         | Office Hours                                                                                                                                                      |
+
+### Week 3, {DATES_WEEK_3}
+
+| 🤝 Developer pairings | 🖥 Assigned Github issue                                                                                                |
+| --------------------- | ----------------------------------------------------------------------------------------------------------------------- |
+| {DEV_3} & {DEV_1}     | [6. As a user, I want to be able to invite others to an existing shopping list.]({REPO_URL}/issues/6)                   |
+| {DEV_4} & {DEV_2}     | [7. As a user, I want to filter my shopping list to make it easier to locate an item in the list.]({REPO_URL}/issues/7) |
+
+| 📖 Learning modules                                    | Description |
+| ------------------------------------------------------ | ----------- |
+| [How to tell your story](https://youtu.be/Zhtd-DdTwSs) |             |
```

Comment:
**Possible Description:** In this video, you'll learn how to share your story with potential employers, regardless of your journey to code.